Mathlib/Algebra/Polynomial/CoeffMem.lean

```diff
@@ -16,8 +17,10 @@ Precisely, we show that each summand needs at most one coefficient of `p` and `d
 of `q`.
 -/
 
+open Function
+
```

Comment:
If you are not using this in a subsequent PR, you could open the namespace in the lemma only.